app/controllers/samples_controller.rb

```diff
@@ -470,7 +470,7 @@ def bulk_import
         if @samples.present?
           render json: { samples: @samples }
         else
-          render json: { status: "No samples imported under #{@bulk_path}. Files must have extension fastq.gz/fq.gz/fastq/fq/fasta.gz/fa.gz/fasta/fa." }, status: :unprocessable_entity
+          render json: { status: "Sorry, we couldn’t find any valid samples in this s3 bucket. Remember, IDseq needs to already have permissions to this bucket. Also, double-check the way you named the files and their extensions. Find the details about permissions and file format requirements by clicking on the \"More Info\" link." }, status: :unprocessable_entity
```

Comment:
I worry that this error message is a little long. Maybe:

Sorry, we couldn’t find any valid samples in this s3 bucket. Please click the \"More Info\" link above for more instructions.

Reply:
@MarkAZhang Let's validate that with Jenn, since this message is her suggestion.

@happyimadesignr Hey Jenn, could you give your feedback on Mark's comments? Thanks.

Reply:
We can always refer to the new documentation page! :)

Reply:
haha it is a bit wordy now that I read it again! 

How about this, shorter but still with some more context:
"Sorry, we couldn’t find any valid samples in this s3 bucket. There may be an issue with permissions or the file format. Click the “More Info" link above for more detailed instructions."

Reply:
That sounds good!

Reply:
Woohoo! @davidrissato